MinecraftClient/ChatBot.cs

```diff
@@ -984,12 +984,23 @@ protected Mapping.Location GetCurrentLocation()
         /// <param name="location">Location to reach</param>
         /// <param name="allowUnsafe">Allow possible but unsafe locations thay may hurt the player: lava, cactus...</param>
         /// <param name="allowDirectTeleport">Allow non-vanilla teleport instead of computing path, but may cause invalid moves and/or trigger anti-cheat plugins</param>
+        /// <param name="allowApproachIfGoalNotReached">If no valid path can be found return a path that approaches your goal</param>
+        /// <param name="mindistance">A minimal distance you want to keep to your goal</param>
```

Comment:
If implementing `maxOffset`, I'd suggest changing this parameter to `minOffset`.
```suggestion
        /// <param name="minOffset">Do not get closer of destination than specified distance.</param>
```
Note that `maxOffset` must be `>= minOffset` so an ArgumentException may be used here.